lib/src/store/moor/moor_store.dart

```diff
@@ -549,7 +547,7 @@ class MoorStore extends Store {
           invitedMembersCount: roomRecord.summaryInvitedMembersCount,
         ),
         directUserId: roomRecord.directUserId != null
-            ? UserId(roomRecord.directUserId ?? '')
+            ? UserId(roomRecord.directUserId!)
```

Comment:
What if we use just UserId(roomRecord.directUserId ?? '') ?

Reply:
We need to pass null if directUserId is null, not UserId("")

Reply:
Okey, yes
I agree